test/WebJobs.Script.Tests/CSharpEndToEndTests.cs

```diff
@@ -32,6 +32,25 @@ public CSharpEndToEndTests(TestFixture fixture) : base(fixture)
         }
 
         [Fact]
+        public async Task NotificationHub()
+        {
+            await NotificationHubTest();
+        }
+
+        [Fact]
+        public async Task NotificationHub_Out_Notification()
+        {
```

Comment:
Regarding the TouchProjectJson - why is that needed? If a function includes a project.json, it should be discovered on startup - I.e., I don't see why you need ScriptHostManager + file watching for these tests.